.circleci/config.yml

```diff
@@ -147,8 +147,28 @@ jobs:
           environment:
             NODE_ENV: production
 
+  messaging-monitor:
+    docker:
+      - *docker_image
+    parameters:
+      signal-server:
+        type: string
+    steps:
+      - checkout
+      - run:
+          command: npm run start
+          working_directory: ./tools/monitors/messaging-monitor
+          environment:
+            SIGNAL_SERVER: << parameters.signal-server >>
+
+parameters:
+  run_daily_workflow:
```

Comment:
do we then call https://circleci.com/docs/scheduled-pipelines/#use-the-api to schedule the workflow?

Reply:
yeah, or in the ui, I've set that up now